### ISSUE TITLE
A quession  about D_net!

### ISSUE THREAD

**Issue by Wuuu3511** [baegwangbin/MaGNet#11]
thanks for your nice work!
I trained the D_net on DTU dataset, the training loss  declined normally(avg depth_error 5mm), but on the validation datase, the loss is high(avg depth_error 50mm). That seems like an overfit,.Could you tell me how to deal with this problem?

**Comment by baegwangbin** [comment]
Hi, D-Net is trained to estimate the depth map in absolute scale. If the scale of the test scene is significantly different from the scale of the training scenes, the depth error is likely to be high. 

You can try performing some sort of scale-matching to minimize the reprojection error.

MaGNet aims to tackle multi-view depth estimation tasks where the training scenes and test scenes have similar metric scales. For example, if you need a depth estimation method for indoor scenes, you can assume that the scale of the training/test scenes will more or less be similar. Under such a scenario, it becomes useful to utilize monocular cues.

**Comment by Wuuu3511** [comment]
> Hi, D-Net is trained to estimate the depth map in absolute scale. If the scale of the test scene is significantly different from the scale of the training scenes, the depth error is likely to be high.
> 
> You can try performing some sort of scale-matching to minimize the reprojection error.
> 
> MaGNet aims to tackle multi-view depth estimation tasks where the training scenes and test scenes have similar metric scales. For example, if you need a depth estimation method for indoor scenes, you can assume that the scale of the training/test scenes will more or less be similar. Under such a scenario, it becomes useful to utilize monocular cues.

Thanks for your reply！
Every scene in DTU dataset has seem depth range(425mm--900mm).Does this mean that every scene has the same scale? Is there a relationship between depth range and scale?
